packages/battery/battery/pubspec.yaml

```diff
@@ -16,13 +16,15 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
-  battery_platform_interface: ^1.0.0
+  meta: ^1.3.0-nullsafety.6
+  battery_platform_interface:
+    git:
+      url: https://github.com/bparrishMines/plugins.git
+      ref: batter_interface_nnbd
+      path: packages/battery/battery_platform_interface
 
 dev_dependencies:
-  async: ^2.0.8
-  test: ^1.3.0
-  mockito: ^4.1.1
+  mockito: ^5.0.0-nullsafety.0
```

Comment:
same. just `^5.0.0-nullsafety` should be enough

Reply:
It looks like mockito's min is `5.0.0-nullsafety.0`